tempest2/src/test/kotlin/app/cash/tempest2/LogicalDbBatchTest.kt

```diff
@@ -62,6 +62,29 @@ class LogicalDbBatchTest {
     assertThat(loadedItems.getItems<PlaylistInfo>()).containsExactly(playlistInfo)
   }
 
+  @Test
+  fun `batchLoad greater than max batch size`() = runBlockingTest {
+    val albumTracks = (1..105).map {
```

Comment:
here and elsewhere in the related tests suggest using the hoisted `MAX_BATCH_READ` / `MAX_BATCH_WRITE` constants with some `+ N` expression to create requests that exceed the configured max page size